src/liberror/specification

```diff
@@ -648,3 +648,27 @@ severity:normal
 ingroup:plugin
 module:storage
 see:9 75 109
+
+number:111
+description:Libgcrypt failed to initialize due to a version mismatch.
```

Comment:
also here you can easily abstract from libgcrypt. libgcrypt can be mentioned in the reason (e.g. "libgrypt 11 != libgrypt 20")